src/Modules/Admin.php

```diff
@@ -129,6 +129,15 @@ public function render_menu(): void {
 			<?php if ( is_array( $variables ) ) : ?>
 				<h3><?php esc_html_e( 'Environment Variables', 'snapwp-helper' ); ?></h3>
 				<p><?php esc_html_e( 'These `.env` variables are used by SnapWP\'s frontend to connect with your WordPress backend.', 'snapwp-helper', ); ?></p>
+				<p>
+					<?php
+					printf(
+						// translators: %s is the hyperlink to the 'Config API & Environment Variables' doc.
+						esc_html__( 'Need help setting up environment variables? Refer to the %s doc.', 'snapwp-helper' ),
+						'<a href="https://github.com/rtcamp/snapwp/blob/develop/docs/config-api.md#env-variables" target="_blank">' . esc_html__( 'Config API and Environment Variables', 'snapwp-helper' ) . '</a>'
+					);
```

Comment:
This should have been like this
```suggestion
					printf(
	'%s <a href="https://github.com/rtcamp/snapwp/blob/develop/docs/config-api.md#env-variables" target="_blank">%s</a>',
	esc_html__( 'Need help setting up environment variables? Refer to the %s doc.', 'snapwp-helper' ),
	esc_html__( 'Config API and Environment Variables', 'snapwp-helper' )
);
```

Reply:
Same goes for all the other printf

Reply:
@SH4LIN go ahead an open a PR